### PR TITLE
Added kube2iam chart that sets appropriate values for kops

### PR DESCRIPTION
**Pull request by goruha**
## What 
* Added kube2iam umbrella chart

## Why 
*  To fix configs useful for kops k8s installation

### CODE REVIEW

File: incubator/kube2iam-umbrella/values.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+kube2iam:
+  extraArgs:
+    base-role-arn: *IAM_BASE_ROLE_ARN
+    default-role: *IAM_DEFAULT_ROLE
+  #   api-server: ...
+  #   api-token: ...
+
+  host:
+    ip: $(HOST_IP)
```
Comment (osterman) [inline incubator/kube2iam-umbrella/values.yaml]: what evaluates this?

**Comment by osterman** [comment]
maybe we should rename it to `kops-kube2iam` or `kube2iam-kops` rather than `kube2iam-umbrella`? The `umbrella` doesn't convey useful information.

File: incubator/kube2iam-umbrella/values.yaml (hunk 1 above)
Comment (goruha) [inline incubator/kube2iam-umbrella/values.yaml]: This is just a string that makes to use ENV var HOST_IP that is specified by K8S

**Comment by osterman** [comment]
@goruha see inline comments. Not all addressed.

**Pull request merged**